### PR TITLE
Add netx getting started guide

### PR DESCRIPTION
**Pull request by jwebs1**

### CODE REVIEW

File: Network/netx-getting-started-guide.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
+{{{
```
Comment (chrislittle) [inline Network/netx-getting-started-guide.md]: does this replace https://www.ctl.io/knowledge-base/network/getting-started-with-netx/ ?  

if so your pull request should delete this old kb.

what about https://www.ctl.io/knowledge-base/network/netx-locations-and-endpoints/ ?

Comment (jwebs1) [inline Network/netx-getting-started-guide.md]: @chrislittle old post deleted in the PR...not sure what you're asking about the locations and endpoints KB?

Comment (chrislittle) [inline Network/netx-getting-started-guide.md]: @jwebs1 just making sure the endpoint locations/kb is 100% accurate still that's all.

Comment (jwebs1) [inline Network/netx-getting-started-guide.md]: @chrislittle ah, no but should be replaced soon (if it hasn't already been deleted in another PR)

**Pull request closed without merging**